### DISCUSSION TITLE
How do I open the commit graph view?

### DISCUSSION THREAD

**Discussion by csntr** [jesseduffield/lazygit#3311] [Q&A] [answered]
I went through the menu of every panel and I couldn't find how to view the whole repo in that nice graph as it is shown in the features section of the github page. I was on latest lazygit 0.40.2

**Comment by stefanhaller** [comment, marked as answer]
It's in the "View log options" menu (`c-l`) in the commits panel.

Note that lazygit currently doesn't remember the value when you change it there. If you always want to have the graph on, add 
```
git:
  log:
    showGraph: always
```
to your lazygit config file.

There is a PR open to get rid of the config and have lazygit remember the value (#3197), but unfortunately it has stalled.

BTW, personally I feel the graph should be shown by default; what's the reason again why it isn't, @jesseduffield?

**Reply by csntr** [reply]
nice thank you! 🙏

**Reply by jesseduffield** [reply]
The default is to show the graph when the commits view is maximised. The reason for this is that some repos have graphs that take up a load of space. For example, here's kubernetes:

With graph
![image](https://github.com/jesseduffield/lazygit/assets/8456633/a097f161-2e4e-437b-930a-1b5b28b8548a)

Without graph
![image](https://github.com/jesseduffield/lazygit/assets/8456633/8af18e1b-adad-47ad-9313-c84d0f1234c8)

But, given that both @stefanhaller and myself have set our own config to always show the graph, and given you're asking how to enable this @csntr, we should just always show the graph by default

**Reply by byteZorvin** [reply]
How to maximise the commits view

**Reply by stefanhaller** [reply]
By pressing `+`. But note that the comments above are out of date, we now show the graph always, not only when maximized (by default, that is; you can still change this with a config).
